pyproject.toml

```diff
@@ -34,3 +34,7 @@ build-backend = "hatchling.build"
 
 [tool.pytest.ini_options]
 asyncio_default_fixture_loop_scope = "session"
+
```

Comment:
(discussed offline) Need a python version bump at the top of this file.

Reply:
Yeah I think we can do that in a separate PR anyway, it's unrelated to this PR :slightly_smiling_face:

Reply:
Oh, we already bumped requires-python to 3.11! https://github.com/grafana/mcp-grafana/pull/24/commits/c78e98abf54dcbeefb60a4a33e8aa0d60e114c61 should fix this error for 3.11, whoops.